src/.editorconfig

```diff
@@ -0,0 +1,210 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Don't use tabs for indentation.
+[*]
+indent_style = space
+# (Please don't specify an indent_size here; that has too many unintended consequences.)
+
+# Code files
+[*.{cs,csx,vb,vbx}]
```

Comment:
do we need `vb` and `vbx`?

Reply:
I copied the [.editorconfig](https://github.com/dotnet/roslyn/blob/master/.editorconfig) file used by Roslyn team, as recommended [here](https://www.hanselman.com/blog/EditorConfigCodeFormattingFromTheCommandLineWithNETCoresDotnetFormatGlobalTool.aspx). I think that we can modify this as we see fit for our project.

Reply:
Regarding the .editorconfig plugin for VSCode, I am not sure if it is necessary or not. From my understanding, the Omnisharp tools are handling the integration between VSCode and the editorconfig file with the help of the [ms-vscode.csharp](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp) extension. From my local tests, the integration seems to be working fine, at least for the simple things, like the indent size, the editor will reformat the file according to the editorconfig settings, when saving the file.